### ISSUE TITLE
Parts of the model are missing in bimvie.ws

### ISSUE THREAD

**Issue by miolfo** [opensourceBIM/BIMserver#529]
Hello,

We have an issue with an ifc model in which some parts of the model are not visible in the bimvie.ws 
[model_broken.zip](https://github.com/opensourceBIM/BIMserver/files/980815/model_broken.zip)

Some parts of the model are visible in the 3D view, but some are missing although they can be seen in the project tree. 
![missing_models](https://cloud.githubusercontent.com/assets/5992841/25771244/a63d3ac2-3254-11e7-905f-70a6da000551.png) In the attached image you can see that there is a number of items titled "RAUDOITUS, HAKA", but none of them are actually visible in the 3D view.

**Comment by rubendel** [comment]
Can you please state which version you are using?

**Comment by miolfo** [comment]
Bimserver version is 1.5.69

**Comment by rubendel** [comment]
Can you try a newer version, it looks like this on my version.
<img width="470" alt="screenshot 2017-05-09 16 00 53" src="https://cloud.githubusercontent.com/assets/377978/25854572/c25358f8-34d0-11e7-8249-0e41855e4570.png">

**Comment by miolfo** [comment]
Hi @rubendel,

Sorry, in the first post I forgot to specify that the outer layer in the model is toggled off/hidden, and the "RAUDOITUS, HAKA" -objects should be visible inside the model (inside the object called "PILARI"). I tried the model in the newest version of the bimserver (1.5.74), and the same issue persists. However, I also happened to have an installation from an older version of the Bimserver (1.4.0), and in that version the "RAUDOITUS, HAKA" objects are visible inside the "PILARI" object as they should (see the image below). 

![skreenkappi](https://cloud.githubusercontent.com/assets/5992841/25906488/76d30b98-35ad-11e7-9a30-563f3b049f82.png)
